meta-oe/recipes-multimedia/v4l2apps/v4l2rtspserver_0.3.8.bb

```diff
@@ -0,0 +1,13 @@
+SUMMARY = "V4L2rtsp streaming server"
+LICENSE = "Unlicense"
+LIC_FILES_CHKSUM = "file://UNLICENSE;md5=911690f51af322440237a253d695d19f"
+
+SRC_URI = "gitsm://github.com/mpromonet/v4l2rtspserver.git;branch=master;protocol=https"
+SRCREV = "b12cf350dc573ab0ff9d1b51a21daf3c218b614e"
+
+PV .= "+0.3.8+git${SRCPV}"
```

Comment:
The `SRCREV` is same as [0.3.8](https://github.com/mpromonet/v4l2rtspserver/releases/tag/v0.3.8) release so appending to `PV` here is not required. Please delete this statement.